dev/how-to/deploy/data-migration-with-binary.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Deploy Data Migration Cluster Using DM Binary
+summary: Learn how to deploy the Data Migration cluster using DM binary.
+category: how-to
+---
+
+# Deploy Data Migration Cluster Using DM Binary
+
+<!-- markdownlint-disable MD029 -->
+
+This document introduces how to quickly deploy the Data Migration (DM) cluster using DM binary.
+
+## Preparations
+
+Download the official binary from [here](/dev/reference/tools/download.md#dm-data-migration).
+
+The downloaded files have two subdirectories, `bin` and `conf`. The `bin` directory contains the binary files of DM-master, DM-worker, dmctl and Mydumper. The `conf` directory contains the sample configuration files.
+
+## Sample scenario
+
+Suppose that you are going to deploy a DM cluster based on this sample scenario:
+
++ Two MySQL instances are deployed on two servers.
++ One TiDB instance is deployed on one server (in the mocktikv mode).
++ Two DM-worker nodes and one DM-master node are deployed on three servers.
+
+Here is the address of each node:
+
+| Instance or node        | Server address   |
+| :---------- | :----------- |
+| MySQL1     | 192.168.0.1 |
+| MySQL2     | 192.168.0.2 |
+| TiDB       | 192.168.0.3 |
+| DM-master  | 192.168.0.4 |
+| DM-worker1 | 192.168.0.5 |
+| DM-worker2 | 192.168.0.6 |
+
+Enable the binlog on MySQL1 and on MySQL2. DM-worker1 replicates the data from MySQL1 and DM-worker2 replicates the data from MySQL2.
+
+Based on this scenario, the following sections of this document describe how to deploy the DM cluster.
+
+### Deploy DM-worker
+
+Establish the connection between DM-worker and the upstream MySQL instances, and for safety reasons, you must configure the encrypted password.
+
+Encrypt the MySQL password by executing the following command. Suppose the password is "123456".
+
+{{< copyable "shell-regular" >}}
+
+```bash
+./bin/dmctl --encrypt "123456"
+```
+
+Then, you get the encrypted password as shown below. Record this encrypted value, which is used for deploying DM-worker in the following steps.
+
+```
+fCxfQ9XKCezSzuCD0Wf5dUD+LsKegSg=
+```
+
+You can deploy DM-worker by using command-line parameters or by using the configuration file.
```

Comment:
support two ways to set config, but  can't delpoy DM-worker only by command-line, some config can only set in config file